### ISSUE TITLE
Add support for HTML output

### ISSUE THREAD

**Issue by aayushmau5** [asyncapi/diff#85]
This library currently supports JSON, & YAML output. 

We need to support HTML output.

Check https://github.com/asyncapi/diff/issues/75

### The format of HTML output

The goal is to generate a HTML(or jsx?) that can be reused in [AsyncAPI studio](https://github.com/asyncapi/studio) as well. Thus, we need to have a detailed discussion on the format.

**Comment by Khadeeejah** [comment]
@aayushmau5  hello, iwould like to work on this

**Comment by aayushmau5** [comment]
@Khadeeejah Awesome! Issue is yours, but first, I think we need to have a detailed discussion on what the HTML should look like 🤔

**Comment by aayushmau5** [comment]
> The goal is to generate a HTML(or jsx?) that can be reused in AsyncAPI studio as well. Thus, we need to have a detailed discussion on the format.

CC @magicmatatjahu what do you think the format should look like? 

I'll see what I can come up with.

**Comment by Khadeeejah** [comment]
Please let me know what you come up with

**Comment by aayushmau5** [comment]
Sorry for the late reply.

I'm just throwing around some ideas, so feel free to suggest or make some changes.

I think the HTML structure will look very similar to the markdown output. Ex.

```html
<h1>Diff</h1>

<h2>All Breaking Changes</h2>

Changes inside code blocks?

<p><span bold>Action :</span>Add</p>
<p><span bold>Path :</span>/some/path</p>
<p bold>Before :</p>
<code syntaxHighlight>
  {
  Hello: "world"
  }
</code>
<p bold>After :</p>
<code syntaxHighlight>
  {
  Hello: "world"
  }
</code>


<h2>All Non-Breaking Changes</h2>
....

<h2>All Unclassified Changes</h2>
....
```

Some features may be:
- Syntax highlighting in code block(not sure what approach we are taking for styling though)
- Instead of having `After` and `Before`, we can perhaps have the changes in a single code blocks with highlights what's changed. Just like github does in diff output.

This is what I came up with.

My current question is what approach are we taking to styling?(Should we include style sheets in the output, having class names in html structure so users can make changes based on class names, some other approach?)

CC @derberg Need some suggestions here :)

**Comment by derberg** [comment]
sorry @aayushmau5 but I'm the worst person to ask for styling feedback 😄 
what I know though is that I like to copy from others if I do not have an idea on my own 😄

**Comment by sambhavgupta0705** [comment]
Hi @aayushmau5 is this issue still active?

**Comment by aayushmau5** [comment]
@sambhavgupta0705 yes! You want to take this up? Let me know if you need any help.

**Comment by sambhavgupta0705** [comment]
yeahh @aayushmau5 I need some more clarity on this issue.

**Comment by BarathElangovan** [comment]
Hi @aayushmau5 is this issue still active? i can contribute.

**Comment by imabp** [comment]
@aayushmau5 do you need a static html file, which can be displayed on studio? I want to understand the requirement here.

**Comment by imabp** [comment]
In my mind, I want to have something like this. This will essentially, take care of generate of static html file as an asset, which can be used in CIs for. visualization or in studio. What are your thoughts? @aayushmau5 

`cli generate-diff  document1 document2 --html --path "/"`
